docs/packaging/creating-a-new-package.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Creating a New Package
+summary: Creating a New Package
+---
+
+# Creating a New Package
+
+This page will take you through the step required to build a new package on Solus. For this example, we will use the command line utility [tree](https://gitlab.com/OldManProgrammer/unix-tree), which is already packaged for Solus, but we will pretend like it's not.
+
+## Create a new Package Directory
+
+From within your build directory, create a new directory for the new package with the package name , then create the required Makefile within that directory:
+
+```bash
+mkdir tree
+cd tree
+echo "include ../Makefile.common" > Makefile
+```
+
+## Create the `package.yml` File
+
+Create a skeleton `package.yml` file using the `yauto.py` script from `common` You will need a link to the source tarball.   
+You can, of course, create your `package.yml` file entirely from scratch, but the script will give you a better starting point.
+
+```bash
+../common/Scripts/yauto.py https://gitlab.com/OldManProgrammer/unix-tree/-/archive/2.1.1/unix-tree-2.1.1.tar.gz
+```
+
+Consider creating an alias for this script in your `.bashrc` file.
+
+```bash
+alias fetchYml="$HOME/solus-builds/common/Scripts/yauto.py"
+```
+
+Now you should have a `package.yml` file that looks something like this:
+
+```text
+name       : unix-tree
+version    : 2.1.1
+release    : 1
+source     :
+    - https://gitlab.com/OldManProgrammer/unix-tree/-/archive/2.1.1/unix-tree-2.1.1.tar.gz : bcd2a0327ad40592a9c43e09a4d2ef834e6f17aa9a59012a5fb1007950b5eced
+license    : GPL-2.0-or-later # CHECK ME
+component  : PLEASE FILL ME IN
+summary    : PLEASE FILL ME IN
+description: |
+    PLEASE FILL ME IN
+builddeps  :
+setup      : |
+    %configure
+build      : |
+    %make
+install    : |
+    %make_install
+```
```

Comment:
Could this use the YAML code syntax highlighting, or is there a reason that it's using `text`?